scripts/setup.sh

```diff
@@ -21,12 +21,13 @@ case "$1" in
         ;;
 esac
 
-go get github.com/DATA-DOG/godog/cmd/godog
-if $go
+
+if ! $go
```

Comment:
Am I reading this right, that the script calls `go get go-algorand-sdk` if `--go` is _not_ supplied, and skips `go get go-algorand-sdk` if it is supplied?

Reply:
Yes; because setup is called from sdkupdate, and specifying some flag tells the script that that's the one that we already have a local version of.